tests/software/kallistobustools/ref/test.yml

```
@@ -1,38 +1,53 @@
 - name: kallistobustools ref test_kallistobustools_ref_standard
   command: nextflow run tests/software/kallistobustools/ref -entry test_kallistobustools_ref_standard -c tests/config/nextflow.config
   tags:
+    - kallistobustools_ref
+    - kallistobustools_ref_standard
     - kallistobustools
-    - kallistobustools/ref
   files:
     - path: output/kallistobustools/cdna.fa
-      md5sum: da194221c883656c68e2f90d8f77a56b
+      md5sum: 0de7fdac9e7418576987ed93640927c7
     - path: output/kallistobustools/kb_ref_out.idx
-      md5sum: c0ffe9554cb6fdcc7cbb4a1285ca8f29
+      md5sum: 7d5cd0731e7c467e5521d761f7d79895
     - path: output/kallistobustools/t2g.txt
-      md5sum: 3089d303d9cd1a39a27969eca2dcaba7
+      md5sum: d5bcfd61ff078aa92e576abf8cc4fc08
 
 - name: kallistobustools ref test_kallistobustools_ref_lamanno
   command: nextflow run tests/software/kallistobustools/ref -entry test_kallistobustools_ref_lamanno -c tests/config/nextflow.config
   tags:
+    - kallistobustools_ref
+    - kallistobustools_ref_lamanno
```

```suggestion
    - kallistobustools/ref
```

we have changed the format of the test tags, going only with `tool` and `tool/subtool` now for all tests